exec/java-exec/src/main/java/org/apache/drill/exec/planner/physical/explain/NumberingRelWriter.java

```diff
@@ -47,6 +50,7 @@
   private final SqlExplainLevel detailLevel;
   protected final Spacer spacer = new Spacer();
   private final List<Pair<String, Object>> values = new ArrayList<>();
+  private final Map<String, Prel> registry;
```

Comment:
The term 'registry' seems somewhat generic since this functionality is only used for the special case of Correlate-Unnest.  Perhaps sourceOperatorRegistry or simply sourceOperatorMap ?